### PR TITLE
Feature/add event style formatter

### PR DESCRIPTION
**Pull request by defond0**
Add in an event formatter for cloudwatch.

### CODE REVIEW

File: marquee/formatter.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 import json
 from logging import Formatter
 
-
```
Comment (theherk) [inline marquee/formatter.py:5]: This line you removed still belongs. Prefer 2 lines between top level functions and classes. Just linting stuff.

File: marquee/formatter.py [hunk 2]
```diff
@@ -25,3 +24,23 @@ def format(self, record):
             'level_number': record.levelno,
             'message': record.msg
         })
+
+class MarqueeEventFormatter(MarqueeFormatter):
+    def __init__(self, event_type=None, source=None, *args, **kwargs):
+        """marquee event formatter.
+        """
+        self.source = source if source else __name__
```
Comment (theherk) [inline marquee/formatter.py]: `is not None`?

File: marquee/formatter.py (hunk 1 above)
Comment (defond0) [reply marquee/formatter.py:5]: got that one switched up, coming right up

File: marquee/formatter.py (hunk 2 above)
Comment (defond0) [reply marquee/formatter.py]: it shall be done

**Comment by theherk** [comment]
Looks good to me.

**Comment by defond0** [comment]
Should be nice and linted up

**Pull request merged**